os/kernel/debug/Make.defs

```diff
@@ -20,7 +20,10 @@ ifeq ($(CONFIG_DEBUG_SYSTEM),y)
 CSRCS += sysdbg.c
 endif
 
-CSRCS += memdbg.c dbg_termination_info.c
+CSRCS += dbg_termination_info.c
+ifeq ($(CONFIG_DEBUG_MM_WARN),y)
```

Comment:
I think verbose(info) is correct for this
Usually WARN level is for potentially harmful situations.

Reply:
Yes, but we used warn intentionally. Because mostly info is not enabled, but we want to check the information for debug.

Reply:
warn is higher level than info, so if warn is enabled, info also need to be enabled. Most of platform follows below.
trace->debug->info->warn->error->fatal

Reply:
Not for us and actually opposite way.
Fatal -> error -> warning ....
Because lower level requires more memory space.